test/cli/data/gapic/api/artman_pubsub.yaml

```diff
@@ -17,37 +17,56 @@ common:
   gapic_api_yaml:
     - ${REPOROOT}/googleapis/google/pubsub/v1/pubsub_gapic.yaml
   output_dir: ${REPOROOT}/artman/output
-  git_repo:
-    # FIXME: Change into the main repo after done with testing.
-    location: git@github.com:lukesneeringer/api-client-staging.git
+  git_repos:
+    staging:
+      location: git@github.com:googleapis/api-client-staging.git
   enable_batch_generation: True
-java:
-  gapic_code_dir: ${REPOROOT}/google-cloud-java/google-cloud-pubsub
-  git_repo:
-    gapic_subpath: generated/java/google-cloud-pubvub-v1
-python:
-  gapic_code_dir: ${REPOROOT}/artman/output/gapic-google-cloud-pubsub-v1
-  git_repo:
-    gapic_subpath: generated/python/gapic-google-cloud-pubsub-v1
-    grpc_subpath: generated/python/proto-google-cloud-pubsub-v1
-go:
-  gapic_code_dir: ${REPOROOT}/gapi-pubsub-go
-  git_repo:
-    gapic_subpath: generated/go/google-pubsub-v1
 csharp:
   gapic_code_dir: ${REPOROOT}/artman/output/gcloud-csharp-pubsub
+go:
+  gapic_code_dir: ${REPOROOT}/gapi-pubsub-go
+  git_repos:
+    go:
+      gapic_subpath: pubsub/apiv1
+      location: git@github.com:GoogleCloudPlatform/google-cloud-go.git
+    staging:
+      gapic_subpath: google-pubsub-v1/cloud.google.com/go/pubsub/apiv1
+java:
+  gapic_code_dir: ${REPOROOT}/artman/output/java/google-cloud-pubsub
+  git_repos:
+    java:
```

Comment:
Honestly, I am happy to let each language lead decide what to name it, and I do not feel they have to match. I plan to send in a googleapis commit so that I am not forcing other people to do the work to accomodate one of my changes,  but I have zero issue if you want to change it.

I am totally happy to name Java's whatever you ask. :-)

For me, personally, I would like them to be named `python` and `nodejs`, because I _do_ work in two languages.